addons/website/static/src/js/menu/content.js

```diff
@@ -56,6 +56,16 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
                 },
             });
         }
+        buttons.push({
+            text: _t("Duplicate Page"),
+            icon: 'fa-clone',
+            classes: 'btn-link float-right',
```

Comment:
```suggestion
            classes: 'btn-link ml-auto',
```
and also delete `float-right` a few lines bellow on `Delete Page` button. It is not working anymore but I am not sure when it brokes. That's the opportunity to fix it